virt/post_installation_configuration/virt-post-install-config.adoc

```diff
@@ -12,7 +12,7 @@ The following procedures are typically performed after {VirtProductName} is inst
 
 * xref:../../virt/post_installation_configuration/virt-node-placement-virt-components.adoc#virt-node-placement-virt-components[Node placement rules for {VirtProductName} Operators, workloads, and controllers]
 
-* xref:../../virt/post_installation_configuration/./.virt-network-config.adoc#virt-post-install-network-config[Network configuration]:
+* xref:../../virt/post_installation_configuration/virt-post-install-network-config.adoc#virt-post-install-network-config[Network configuration]:
```

Comment:
Not sure I understand your question. I evidently made a mistake with this xref. Your change is correct.

Reply:
@apinnick I wanted to make sure the formatting was not intentional, perhaps some formatting I'm not familiar with.

Reply:
@mburke5678  Definitely not intentional! Just my poor, tired, senile brain having hiccups. :-)